cmd/deploy.go

```diff
@@ -108,13 +111,6 @@ func runDeploy(cmd *cobra.Command, _ []string, newClient ClientFactory) (err err
 		if err != nil {
 			return err
 		}
-
-		// update function config if build type has been changed
-		// don't store type `disabled` as it should be used only as a parameter in `func deploy`
-		// and not stored in function config
-		if function.BuildType != config.BuildType && config.BuildType != fn.BuildTypeDisabled {
-			function.BuildType = config.BuildType
-		}
```

Comment:
Why remove this code?

Reply:
That's to avoid the build type to be persisted to the `func.yaml` file